README.md

```diff
@@ -1,24 +1,227 @@
-# README
+<a name="readme-top"></a>
```

Comment:
_note: Since there is no open PR in the frontend project for me to review, I'll include my comments here._

# Frontend review comments

- [ ] Kindly, remove the commented code present in RenderedRooms.js, and ModalRoot.js, in order to improve the repo's cleanliness.

- [ ]  Please, kindly refactor the functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/225207068-f4a4bb46-a679-4cfc-9a81-8e0dcf09a95b.png)